platforms/python/prereqs/build.sh

```diff
@@ -47,6 +47,10 @@ if [ "$debianFlavor" == "stretch" ]; then
     echo "Hack flavor is: "$debianHackFlavor
 
     pythonSdkFileName=python-$PYTHON_VERSION.tar.gz
+
+    if [[ $PYTHON_VERSION == 3.6* ]]; then
```

Comment:
Naive question, but is there a way for us to make this more generic than just targeting 3.6? Please ignore if this problem actually impacts only 3.6, and we don't care about other versions.

Reply:
from the job: 
`ERROR: This script does not work on Python 3.6 The minimum supported Python version is 3.7. Please use https://bootstrap.pypa.io/pip/3.6/get-pip.py instead.`

Reply:
OK, thanks for fixing it!